src/evidently/options/quality_metrics.py

```diff
@@ -0,0 +1,27 @@
+from dataclasses import dataclass
+from typing import Optional, Dict, Tuple, Union
+
+DEFAULT_CONF_INTERVAL_SIZE = 1
+DEFAULT_CLASSIFICATION_THRESHOLD = 0.5
+
+
+@dataclass
+class QualityMetricsOptions:
+    conf_interval_n_sigmas: int = DEFAULT_CONF_INTERVAL_SIZE
+    classification_threshold: float = DEFAULT_CLASSIFICATION_THRESHOLD
+    cut_quantile: Union[None, Tuple[str, float], Dict[str, Tuple[str, float]]] = None
+
+    def as_dict(self):
+        return {
+            "conf_interval_n_sigmas": self.conf_interval_n_sigmas,
+            "classification_threshold": self.classification_threshold,
+            "cut_quantile": self.cut_quantile
+        }
+
+    def get_cut_quantile(self, feature_name: str) -> Tuple[str, float]:
```

Comment:
for mypy fix - set `def get_cut_quantile(self, feature_name: str) -> Optional[Tuple[str, float]]:`